src/include/daos/mem.h

```diff
@@ -50,21 +50,21 @@ typedef uint64_t		umem_off_t;
 /** Number of flag bits to reserve for encoding extra information in
  *  a NULL umem_off_t entry.
  */
-#define UMOFF_NUM_FLAGS		(8)
-/** The absolute value of a flag mask may not exceed this value */
-#define UMOFF_MAX_FLAG		(1ULL << UMOFF_NUM_FLAGS)
+#define UMOFF_NUM_BITS		(3)
```

Comment:
if root is start from offset higher than 256, then I would keep it to 8 because bit-flags can be consumed very soon, although we shouldn't abuse bit-flags

Reply:
Ok, will change that back